### ISSUE TITLE
Build a landing page for the app's root screen.

### ISSUE THREAD

**Issue by BeckyPollard** [badgateway/tt-app#43]
### Task Summary

The current home screen is _cute_ but it could be better.
This is an open-ended task to create a useful homescreen for the time tracker project, maybe something that shows data relevant to the user or some links to exports or something. Maybe some stats?

| Old view |
| ------- |
| ![Screen Shot 2022-09-30 at 3 53 37 PM](https://user-images.githubusercontent.com/52248161/193346654-d90fb2f0-5b66-4297-89e8-05c50881c3f5.png) |

**Comment by evert** [comment]
As a person who's entering time, my homepage should probably be just my timesheet =)

**Comment by BeckyPollard** [comment]
Yeah! This screen should make getting to the meat of the app really easy!
